src/databases/ffp/avtffpFileFormat.C

```diff
@@ -96,6 +102,31 @@ static int InitLibStripack()
 
     // We can't ever close it or we loose access to the functions
     //dlclose(libh);
+#else
+    HINSTANCE libh;
+    if (Environment::exists(envar))
+        libh = LoadLibrary(Environment::get(envar).c_str());
+    if (!libh)
+        libh = LoadLibrary("libstripack.dll");
+    if (!libh)
+        return 1;
+
+    _trmeshp = (trmeshp_t) GetProcAddress(libh, "trmesh");
+   if (!_trmeshp)
+        _trmeshp = (trmeshp_t) GetProcAddress(libh, "trmesh_");
+   if (!_trmeshp)
+        _trmeshp = (trmeshp_t) GetProcAddress(libh, "TRMESH");
+   if (!_trmeshp)
+        _trmeshp = (trmeshp_t) GetProcAddress(libh, "TRMESH_");
+
+    _trlistp = (trlistp_t) GetProcAddress(libh, "trlist");
+   if (!_trlistp)
+        _trlistp = (trlistp_t) GetProcAddress(libh, "trlist_");
+   if (!_trlistp)
+        _trlistp = (trlistp_t) GetProcAddress(libh, "TRLIST");
+   if (!_trlistp)
+        _trlistp = (trlistp_t) GetProcAddress(libh, "TRLIST_");
+#endif
```

Comment:
Is there an `UnloadLibrary` perhaps similar to `dlclose()` on linux. If so, maybe add it but comment it out as above for Linux so anyone stumbling into this code won't decide its an important leak to fix.